psi4/driver/qcdb/molecule.py

```diff
@@ -1354,8 +1366,10 @@ def _raw_to_dict(self, force_c1=False, force_au=False, np_out=True):
         if self.name() not in ['', 'default']:
             molrec['name'] = self.name()
 
-        if force_au:
+        if force_units == 'Bohr':
```

Comment:
I noticed throughout you are not case insensitive for these quantities. Just checking if this is intentional as we are usually not case sensitive for input values.

Reply:
We're case insensitive for user-facing QC functions but not usually for ordinary python fn args, I thought. These are essentially the enums from libmints.Mol so I didn't really consider it. I'll consider bohr/angSTROM over in qcdb, but I'd really rather not get into yes/1/on- and Bohr/au/a.u./bohr-style flexibility.